libs/designsystem/item/src/item.component.ts

```diff
@@ -43,7 +44,14 @@ export class ItemComponent implements AfterViewInit {
   @HostBinding('class.disclosure')
   @Input()
   disclosure: 'link' | 'arrow-more' | 'arrow-down' | 'arrow-up' | null;
-  @Input() selectable: boolean;
+  @Input() set selectable(value: boolean) {
+    this._selectable = value;
+    this.cdr.markForCheck();
+  }
+  get selectable(): boolean {
+    return this._selectable;
+  }
+  private _selectable: boolean;
```

Comment:
Ahh, missed this one before, but consider making this a bit easier to read, and follow the structure from segmented control, theme color directive etc.

```suggestion

  private _selectable: boolean;
  get selectable(): boolean {
    return this._selectable;
  }
  
  @Input() set selectable(value: boolean) {
    this._selectable = value;
    this.cdr.markForCheck();
  }
```